api/k0smotron.io/v1beta1/k0smotroncluster_types.go

```diff
@@ -65,6 +65,12 @@ type ClusterSpec struct {
 	//+kubebuilder:validation:Enum:=kuberouter;calico;custom
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="cniPlugin is immutable"
 	CNIPlugin string `json:"cniPlugin,omitempty"`
+	// Manifests allows to specify a confiMmap with manifests to be
```

Comment:
hmm, could it be a secret instead? I mean in some cases there might be some API tokens or similar in these manifests.

Reply:
That makes sense, I'll change that